packages/commonwealth/client/scripts/views/pages/notification_settings/index.tsx

```diff
@@ -498,6 +504,91 @@ const NotificationSettingsPage = () => {
             </div>
           );
         })}
+      <div style={{ marginTop: 26 }}>
+        <CWText
+          type="h4"
+          fontWeight="semiBold"
+          className="chain-events-section-margin"
+        >
+          Snapshot Subscriptions
+        </CWText>
+        <div className="column-header-row">
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="column-header-text"
+          >
+            Community
+          </CWText>
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="column-header-text"
+          >
+            Email
+          </CWText>
+          <CWText
+            type={isWindowExtraSmall(window.innerWidth) ? 'caption' : 'h5'}
+            fontWeight="medium"
+            className="last-column-header-text"
+          >
+            In-App
+          </CWText>
+        </div>
+      </div>
+      {Object.entries(bundledSnapshotSubs)
+        .sort((x, y) => x[0].localeCompare(y[0]))
+        .map(([chainName, subs]) => {
```

Comment:
So I'm assuming this here is the reason that you included chain in the `SubUniqueData` type and elsewhere but as discussed on Slack, snapshot spaces are not tied to a community/chain so really we should only be displaying the snapshot id and not a community. This is important since 2 different communities could be associated with the same snapshot space so if that happens which chain/community name/icon would you pick?